tests/integration/flows/test_sql_server_to_duckdb.py

```diff
@@ -37,6 +37,10 @@ def create_sql_server_table():
         if_exists="replace",
         credentials=json.loads(credentials_str),
     )
+    drop_sqlserver = AzureSQLDBQuery()
```

Comment:
comment: I wouldn't name this `drop_sqlserver` and this variable is for the generic query task, not specifically drop.